draft-ietf-ice-rfc5245bis.xml

```diff
@@ -673,7 +680,7 @@ controlling agent to nominate a candidate pair.
 
 <t hangText="Nomination, Regular Nomination:"> The process of the
 controlling agent indicating to the controlled agent which candidate
-pair the ICE agents should use for sending and receiving data.
+pair the ICE agents shall use for sending and receiving data.
```

Comment:
shall is also 2119 word. Will?

Reply:
Will fix that.

Reply:
I realized that there are a number of "shall" that cannot be converted to "will". In some cases I actually think they shall be upper-case MUST.